openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -588,7 +588,7 @@ def _create_instances_for_aov(
             self.log.debug("instances:{}".format(instances))
         return instances
 
-    def _get_representations(self, instance, exp_files, additional_data):
+    def _get_representations(self, instance, exp_files):
```

Comment:
If we're changing the signature isn't there a function call somewhere using this which needs to change along too?

Reply:
As far as I could see in the code there, there is only one call of the function, and it was not calling it with third input `additional_data`.  Look to https://github.com/ynput/OpenPype/pull/4552/files#diff-2534ee7134eb5afe9462ddc39bdf86307c14a8592ff7c3deed8246041b7edb89R971 it is the only occurrence in the plugin.

Reply:
👍 So the signature was just wrong before, check.